src/main/java/com/mercadolibre/be_java_hisp_w24_g02/controller/UserController.java

```diff
@@ -1,22 +1,41 @@
 package com.mercadolibre.be_java_hisp_w24_g02.controller;
 
 import com.mercadolibre.be_java_hisp_w24_g02.dao.UserRelationshipsDTO;
+import com.mercadolibre.be_java_hisp_w24_g02.exception.NotFoundException;
 import com.mercadolibre.be_java_hisp_w24_g02.service.interfaces.IUserService;
 import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
 import org.springframework.web.bind.annotation.RestController;
 
 @RestController
 @RequiredArgsConstructor
 public class UserController {
 
     private final IUserService userService;
+
     @GetMapping("/users/{userId}/followers/list")
     public ResponseEntity<UserRelationshipsDTO> getUserFollowers(@PathVariable Integer userId) {
         UserRelationshipsDTO userRelationshipsDTO = userService.getUserFollowers(userId);
         return ResponseEntity.ok(userRelationshipsDTO);
     }
 
+
+    @PostMapping("/users/{userId}/follow/{userIdToFollow}")
+    public ResponseEntity<String> followUser(@PathVariable Integer userId, @PathVariable Integer userIdToFollow) {
+        try {
+            userService.followUser(userId, userIdToFollow);
+            return ResponseEntity.ok("Usuario seguido exitosamente");
```

Comment:
When exceptions are handled through the GlobalController, it is not necessary to use try catch in this part. The method like this would suffice:
`
userService.followUser(userId, userIdToFollow);
`
`
return ResponseEntity.ok("Usuario seguido exitosamente");
`

Since in the service you have control by throwing those exceptions and those are caught by the GlobalException.